mgr.go

```diff
@@ -29,6 +29,7 @@ import (
 	ipcLib "github.com/nestybox/sysbox-ipc/sysboxMgrLib"
 	"github.com/nestybox/sysbox-libs/dockerUtils"
 	"github.com/nestybox/sysbox-libs/formatter"
+	"github.com/nestybox/sysbox-libs/idMap"
```

Comment:
> This way sysbox-runc won't skip asking sysbox-mgr to prep mounts when ovfs on
ID-map-mounts is supported.

Is this commit-description line correct?

Reply:
Yes; what happened was the following: normally sysbox-runc asks sysbox-mgr to "prepare" (i.e., chown) host dir/vols mounted into the container's `/var/lib/docker` and similar dirs. However if the host supports overlayfs on ID-mapped mounts, we can now skip the "prepare" step and ID-map the host dir/vol instead. So I initially added code to skip the "prepare" step in that scenario. However, later I realized that the "prep" is not just doing a chown, but also tracking that the same host dir/vol is never mounted into 2 or more sysbox containers at the same time (as otherwise you have two inner dockers sharing the same `/var/lib/docker` for example). 

In the commit message you asked about, I am saying "don't skip the prepare step" so as to catch that situation, but sysbox-mgr will still skip the chown within the prepare step when overlayfs on ID-mapping is supported.